tests/migrations/test_commands.py

```diff
@@ -1427,6 +1427,48 @@ def test_prune_no_app_label(self):
         with self.assertRaisesMessage(CommandError, msg):
             call_command("migrate", prune=True)
 
+    @override_settings(
+        MIGRATION_MODULES={
+            "migrations": "migrations.test_migrations_squashed_no_replaces",
+            "migrations2": "migrations2.test_migrations_2_squashed",
+        },
+    )
+    def test_prune_specified_app_other_irrelevant_app_with_replaces(self):
```

Comment:
This is not a proper regression test because it passes without the patch.

Reply:
The new test is still not a proper regression test as it passes without the patch.

Reply:
@felixxm I've updated the tests. can you review it again?

Reply:
@felixxm can you please review it again?

Reply:
@smoeinbbp Please do not mention people directly when a PR is ready for a re-review, reviewers will get to this when they have some availability. The proper procedure to get a re-review is described in the links I shared in my [previous comment](https://github.com/django/django/pull/17749#issuecomment-1898400321). You need to unset the relevant ticket flags following [these docs](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/submitting-patches/#patch-style) so this PR gets added back to the "patches needing review" section of the [Django Developer Dahsboard](https://dashboard.djangoproject.com/) .